useIsShake.js

```diff
@@ -69,15 +54,13 @@ export function useIsShake() {
   };
 
   useEffect(() => {
-    _subscribe();
-
-    const onLoad = setTimeout(() => {
-      isToggleReady = true;
-      clearTimeout(onLoad);
-    }, SVG_LOAD_DURATION);
-
+    // invocation of _subscribe when isToggleReady === true
+    if (isToggleReady) {
+      _subscribe();
+      // isShake Accelerometer listener is removed when !isToggleReady
+    } else _unsubscribe();
```

Comment:
Debounce design reimplemented here through addition and removal of Accelerometer listener based on the state of isToggleReady.